go.mod

```diff
@@ -13,6 +13,7 @@ require (
 require (
 	github.com/aymerick/douceur v0.2.0 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bxcodec/faker/v3 v3.8.0
```

Comment:
what does faker do? Can you provide some context for the lib?

Reply:
faker contains package slice that I am using for the contains() check to see if array contains an element or not